mixin-utils/utils.libsonnet

```diff
@@ -392,18 +392,16 @@ local g = import 'grafana-builder/grafana.libsonnet';
       for group in groups
     ],
 
-  removeRuleGroup(ruleName):: {
-    local removeRuleGroup(rule) = if rule.name == ruleName then null else rule,
-    local currentRuleGroups = super.groups,
-    groups: std.prune(std.map(removeRuleGroup, currentRuleGroups)),
+  removeRuleGroup(groupName):: {
+    groups: std.filter(function(group) group.name != groupName, super.groups),
   },
 
   removeAlertRuleGroup(ruleName):: {
-    prometheusAlerts+:: $.removeRuleGroup(ruleName),
+    prometheusAlerts+: $.removeRuleGroup(ruleName),
```

Comment:
`+:` means 'keep visiblity as it was, so if the field was hidden upstream, then it will stay hidden.